print/print-apps/oereb/topiclegend.jrxml

```diff
@@ -44,7 +44,7 @@
 				<textElement textAlignment="Right" verticalAlignment="Top" markup="html">
 					<font fontName="Cadastra" size="9"/>
 				</textElement>
-				<textFieldExpression><![CDATA[java.text.MessageFormat.format("{0} m<sup>2</sup>", java.text.NumberFormat.getInstance($P{REPORT_LOCALE}).format($F{Area}))]]></textFieldExpression>
+				<textFieldExpression><![CDATA[java.text.MessageFormat.format("{0} m<sup>2</sup>", String.valueOf($F{Area}))]]></textFieldExpression>
```

Comment:
If we decide to do the rounding (the rendering) out of the template they should already be as string, here we have a special case with integer where we have no artifacts, but if we move to one decimal we can have some artifacts because number are coded in binary, not in decimal...